doc/requirements.txt

```diff
@@ -45,3 +45,4 @@ jinja2<3.1
 parmed<=3.4.4; python_version<"3.8"
 dask==2022.2.0
 polars
+geoparse<=2.0.3
```

Comment:
Do you know which `geoparse` version used before?

Reply:
The last successful build used 2.0.3